docs/MigrationGuide.md

```diff
@@ -16,6 +16,7 @@ To see the biggest differences please consult the [changelog](Changelog.md).
   to implement it only if your DM will be wrapped by `PeriodicDeploymentManager`
 
 ### REST API changes
+* [#4697](https://github.com/TouK/nussknacker/pull/4697) Use processingType instead of processName for valid nad suggestions endpoints
```

Comment:
And add scenarioName parameter to validate resource. Also, please use the full name of endpoint instead of "valid"

Reply:
Thanks, changed :+1: